tests/Util/yaml_fixtures/a_wip.test

```diff
@@ -0,0 +1,13 @@
+security:
+  firewalls:
+#  main:
+===
+$data['security'] = [
+    'firewalls' => 'a firewall',
+    'providers' => 'a provider'
```

Comment:
ysm has the wrong indentation _i think_ for `providers` - it looks like its trying to set it at `0` when it should be `2`?